.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "\U00002753 Crystal Community Forum"
+    url: https://forum.crystal-lang.org/c/help-support
+    about: Please ask and answer questions here.
+  - name: "\U0001F4AC Crystal Community Chat"
+    url: https://gitter.im/crystal-lang/crystal
+    about: "Official Gitter community chat, also available on IRC: http://webchat.freenode.net/?channels=#crystal-lang"
```

Maybe:
```suggestion
    about: Get in touch with the community. If you're unsure about which kind of issue report you should file the chat can help you quickly.
```
I don't think we need the IRC link here, especially when it doesn't workas a link anyway. Could be `(IRC: #crystal-lang on freenode)` or maybe just omit it entirely.

---

Feels a bit too long too.

---

Same here, I also pushed an alternative sentence, inspired by yours.